integration-tests/contract-with-txr1-address.spec.ts

```diff
@@ -15,7 +15,7 @@ CONFIGS().forEach(({lib, setup, protocol}) => {
   describe('test', () => {
     type Storage = { addressSet: string[], addressMap: MichelsonMap<string, number>; }
 
-    jakartanet('should pass with tz addresses', async (done) => {
```

Comment:
Should these tests run on all testnets? Using `it` rather than `limanet`?
I am not too sure why they were not running on Kathmandu..